dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/audit/BasicAuthAuditAUTHZTest.java

```diff
@@ -83,6 +89,13 @@ public class BasicAuthAuditAUTHZTest {
     // (hopefully) be reflected in the static references as well.
     private static TestName _name = new TestName();
 
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat("com.ibm.ws.webcontainer.security.fat.basicauth.audit",
+    EERepeatActions.EE10, // EE10
+    EERepeatActions.EE9, // EE9
+    EERepeatActions.EE8).with(new EmptyAction().fullFATOnly())
+    .andWith(new FeatureReplacementAction(AuditCommonTest.AUDIT_1_0_FEATURES, AuditCommonTest.AUDIT_2_0_FEATURES));
```

Comment:
```suggestion
    /**
     * Need the first repeat to make sure that audit-2.0 from a previous repeat gets put back to audit-1.0
     */
    @ClassRule
    public static RepeatTests auditRepeat = RepeatTests.with(new FeatureReplacementAction("audit-2.0", "audit-1.0"))
                    .andWith(new FeatureReplacementAction("audit-1.0", "audit-2.0"));
```
Don't need the EE repeat since it is already done in FATSuite.